app/controllers/api/v1/reviews_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V1::ReviewsController < ApplicationController
-
```

Comment:
skip_before_action :verify_authenticity_token

Should be able to allow you to remove those headers